### PR TITLE
Security maintenance update to v0.10.42

### PR DESCRIPTION
**Pull request by abtris**
https://nodejs.org/en/blog/vulnerability/february-2016-security-releases/

### CODE REVIEW

**Comment by abtris** [comment]
No tests for this.

File: coreapp-dev/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 FROM        apiaryio/base-dev-nodejs
 MAINTAINER  Apiary <sre@apiary.io>
 
-ENV REFRESHED_AT 2010-02-05
+ENV REFRESHED_AT 2010-02-10
```
Comment (w-vi) [inline coreapp-dev/Dockerfile]: Shouldn't the date be same as others?

Comment (abtris) [inline coreapp-dev/Dockerfile]: Yes, but this change was only for refresh this image.

Comment (w-vi) [inline coreapp-dev/Dockerfile]: These images are built automatically, so whenever there is a push to repo they get rebuilt, don't they? So there is no need to change the file to trigger the build, am I correct?

Comment (abtris) [inline coreapp-dev/Dockerfile]: I'm not sure if is rebuild if you change parent only.

**Pull request merged**